sparkplug_b/stand_alone_examples/c/template_as_custom_props/example.c

```diff
@@ -209,7 +209,6 @@ void my_log_callback(struct mosquitto *mosq, void *userdata, int level, const ch
 void publish_births(struct mosquitto *mosq) {
 	// Initialize the sequence number for Sparkplug MQTT messages
 	// This must be zero on every NBIRTH publish
-	seq = 0;
```

Comment:
The functionality this `seq` variable provided has been replaced by the function `reset_sparkplug_sequence`.  You can either put a call to that function here, or (preferably) down around line 228 just before the call to `get_next_payload` when building the NBIRTH message.